autocorpus/__main__.py

```diff
@@ -38,14 +35,9 @@
         "expects for the lang argument, default eng"
     ),
 )
-
-group = parser.add_mutually_exclusive_group()
-group.add_argument(
+parser.add_argument(
     "-c", "--config", type=str, help="filepath for configuration JSON file"
 )
```

Comment:
We should keep this mutually exclusive group because it's being used in #149

Reply:
Oops! I'll put it back